### DISCUSSION TITLE
server_config.toml missing from windows install

### DISCUSSION THREAD

**Discussion by LanceLake** [space-wizards/space-station-14#28694] [Questions] [answered]
Yes, before you ask, I did look in the %APPDATA% directory. No directory there.

I did all the steps on here... https://docs.spacestation14.com/en/general-development/setup/server-hosting-tutorial.html for level 1. But no server_config.toml exists anywhere.

**Comment by PJB3005** [comment, marked as answer]
The `server_config.toml` should be next to the `Robust.Server.exe` you extracted from the zip file. Is it not there?

**Reply by LanceLake** [reply]
It was not... Though I initially tried to run the non-robust (Content) (so I can get auto updates) and I was trying to follow along with that one. I have fixed the issue and now, I see it. It doesn't load though. Well, it says it's loading it, but none of the convars are changed from default. So now I put them on the command line.
